endpoints/setuid.go

```diff
@@ -313,14 +306,14 @@ func parseConsentFromGppStr(gppQueryValue string) (string, error) {
 	return gdprConsent, nil
 }
 
-func getSyncer(query url.Values, syncersByKey map[string]usersync.Syncer) (usersync.Syncer, error) {
+func getSyncer(query url.Values, syncersByBidder map[string]usersync.Syncer) (usersync.Syncer, error) {
 	key := query.Get("bidder")
 
 	if key == "" {
 		return nil, errors.New(`"bidder" query param is required`)
 	}
 
-	syncer, syncerExists := syncersByKey[key]
+	syncer, syncerExists := syncersByBidder[key]
```

Comment:
Should `key` within the brackets `[key]` now be called `bidder`?

Reply:
Yes, this change is already present in this PR: https://github.com/prebid/prebid-server/pull/2897#discussion_r1267305253
But yes, it will be simpler to have it here :)